instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/RedisCommandNormalizer.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.lettuce.v5_1;
+
+import static java.util.Collections.unmodifiableMap;
+
+import io.opentelemetry.javaagent.instrumentation.lettuce.v5_1.RedisCommandNormalizer.CommandNormalizer.CommandAndNumArgs;
+import io.opentelemetry.javaagent.instrumentation.lettuce.v5_1.RedisCommandNormalizer.CommandNormalizer.Default;
+import io.opentelemetry.javaagent.instrumentation.lettuce.v5_1.RedisCommandNormalizer.CommandNormalizer.Eval;
+import io.opentelemetry.javaagent.instrumentation.lettuce.v5_1.RedisCommandNormalizer.CommandNormalizer.MultiKeyValue;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/** This class is responsible for masking potentially sensitive data in Redis commands. */
+public final class RedisCommandNormalizer {
```

Comment:
This class can be reused by other Redis libraries' instrumentations; I'll move it to some common package (`javaagent-api`?) in next PR and call it whenever args are available. Currently lettuce 5.1 is the only Redis lib instrumentation that logs actual statement, not just the bare command.

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1406